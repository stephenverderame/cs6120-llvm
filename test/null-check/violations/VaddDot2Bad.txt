[31mSafety Violation[00m: Use of potentially null pointer in store at VaddDot2Bad.cc:27
[31mSafety Violation[00m: Use of potentially null pointer in store at VaddDot2Bad.cc:30
clang++-17: error: clang frontend command failed with exit code 2 (use -v to see invocation)
