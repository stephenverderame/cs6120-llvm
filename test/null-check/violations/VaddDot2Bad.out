[31mSafety Violation[00m: Use of potentially null pointer in store at VaddDot2Bad.cc:27
[31mSafety Violation[00m: Use of potentially null pointer in store at VaddDot2Bad.cc:30
